sdk/tests/test_finbourne_identity.py

```diff
@@ -1,18 +1,28 @@
 import unittest
-from finbourne_identity import api as la, ApiClientBuilder
+from finbourne_identity import ApiClientBuilder as identity_api
+from finbourne_identity import api as iapi
 from finbourne_identity.utilities import ApiClientFactory
 
-
 class Lydia(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        api_client = ApiClientFactory()
-        cls.domains_api = api_client.build(la.DomainsApi)
+        api_client_factory = ApiClientFactory(api_secrets_filename="secrets.json")
+        api_client = api_client_factory.api_client
+        identity_api_client = identity_api.build(api_secrets_filename="secrets.json",
+                                                 token=api_client.configuration.access_token)
+        cls.users_api = iapi.UsersApi(identity_api_client)
+        cls.roles_iapi = iapi.RolesApi(identity_api_client)
+
+    def test_roles(self):
+        identity_roles = self.roles_iapi.list_roles()
+        self.assertIsNotNone(identity_roles)
+        self.assertIsNot(identity_roles, str)
+
+    def test_users(self):
+        user_list = self.users_api.list_users()
+        self.assertIsNotNone(user_list[0].to_dict()['id'])
```

Comment:
this will error in a not nice way if `user_list` is empty, its ok to `assertGreater(len(user_list), 0`